crates/sargon/src/radix_connect/wallet_interaction/dapp_wallet_interaction/dapp_to_wallet/dapp_metadata/interaction_metadata.rs

```diff
@@ -37,7 +37,7 @@ impl HasSampleValues for DappToWalletInteractionMetadata {
     fn sample() -> Self {
         Self::new(
             WalletInteractionVersion::sample(),
-            NetworkID::Stokenet,
+            NetworkID::Mainnet,
```

Comment:
These samples were invalid. The dAppDefinigionAddress was in mainnet, but the NetworkID was in Stokenet.